insolar/insolar
# MN-632: add metrics for a gRPC server

**- What I did**
I have registered useful metrics of the gRPC server.
**- How I did it**
I have used the go-grpc-prometheus library, it adds counters and histograms via interceptors.
**- How to verify it**
Run the Insolar network and monitoring, and see new metrics with the grpc_ prefix.

Note:
You can see a full list of new metrics here https://insolar.atlassian.net/browse/MN-632?focusedCommentId=23377
